tests/test_features/test_pcoords.py

```diff
@@ -78,6 +78,19 @@ def test_pcoords_sample_int(self):
         visualizer = ParallelCoordinates(sample=10)
         visualizer.fit_transform(self.X, self.y)
 
+    def test_pcoords_sample_int_shuffle(self):
+        """
+        Assert no errors occur using integer 'sample' argument and shuffle, with different random_state args
+        """
+        visualizer = ParallelCoordinates(sample=3, shuffle=True)
+        visualizer.fit_transform(self.X, self.y)
+
+        visualizer = ParallelCoordinates(sample=3, shuffle=True, random_state=444)
+        visualizer.fit_transform(self.X, self.y)
+
+        visualizer = ParallelCoordinates(sample=3, shuffle=True, random_state=np.random.RandomState())
+        visualizer.fit_transform(self.X, self.y)
```

Comment:
Should we also add a `shuffle=False`?

Reply:
Added test_pcoords_sample_shuffle_false()